spacy/ml/models/tagger.py

```diff
@@ -7,8 +7,21 @@
 
 
 @registry.architectures("spacy.Tagger.v1")
```

Comment:
I think this should probably be moved to `spacy-legacy` instead.

Reply:
Ok, I have also made a draft pull request for `spacy-legacy`: https://github.com/explosion/spacy-legacy/pull/20